provisions/roles/cleanup/openshift/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Remove all containers
+  sudo: yes
+  shell: docker rm -f `docker ps -a -q`
+  ignore_errors: yes
+
+- name: Remove all images
+  sudo: yes
+  shell: docker rmi -f `docker images -a -q`
```

Comment:
Removing all images is also unnecessary. We are not cleaning up docker here, but only openshift. If you have some other reason for doing so, I will be eager to listen.